README.md

```diff
@@ -57,7 +57,9 @@ the set up is pretty simple. Just run:
 ```
 $ mkvirtualenv ansible
 (ansible)$ git clone https://github.com/dimagi/commcare-cloud.git
+(ansible)$ pip install -r commcare-cloud/requirements.txt
```

Comment:
Nice, thanks for adding this!

Could you change this to say

```
pip install pip-tools
pip-sync commcare-cloud/requirements.txt
```

`pip-sync` is a nice tool that in addition to installing everything in `requirements.txt` also uninstalls everything _not_ in `requirements.txt`, which is important for some changes where we're replacing one package with another (e.g. recently we replaced `pycrypto` with a better maintained for, `pycryptodome`).

Reply:
Cool, added here: https://github.com/dimagi/commcare-cloud/pull/1944/commits/74cf79c3fae053a04c1cc76568e0f6937de44c58

Reply:
Could that replace `uninstall-requirements.txt` in hq?